roles/commcare_analytics/templates/rabbitmq/rabbitmq.conf.j2

```diff
@@ -0,0 +1,9 @@
+#
+# {{ ansible_managed }}
+#
+log.channel.level = warning
+log.connection.level = warning
+log.federation.level = warning
+log.mirroring.level = warning
+consumer_timeout = 86400001
+heartbeat = 600
```

Comment:
Just a potential gotcha, celery also sets a [broker_timeout](https://docs.celeryq.dev/en/v5.2.7/userguide/configuration.html#broker-heartbeat), and the lower of that value and [this heartbeat value](https://www.rabbitmq.com/docs/heartbeats#heartbeats-timeout) is what is actually used.